R/FilteredData.R

```diff
@@ -589,6 +620,17 @@ FilteredData <- R6::R6Class( # nolint
       div(
         id = ns(NULL), # used for hiding / showing
         include_css_files(pattern = "filter-panel"),
+        div(
+          id = ns("switch-button"),
+          class = "inline-block",
+          shinyWidgets::switchInput(
+            ns("filter_turn_onoff"),
+            label = "TURN",
+            value = TRUE,
+            inline = FALSE,
+            handleWidth = "80px"
+          )
+        ),
```

Comment:
Please spend more time on UX here. IMO right now it is too maximal

Reply:
I like the current UI as is obvious and the functionality important, please give me your idea then.

Reply:
It's not about the importance of the feature - it's more how it fits into the overall GUI. Right now it doesn't fit well.
Maybe something like below? It's a minimal switch with decreased opacity.
![image](https://user-images.githubusercontent.com/12943682/190375631-d86e62c5-5fa8-4290-adba-32cfce565b13.png)

Reply:
I think the button that @Polkas proposed could work if we label it properly:

![Screen Shot 2022-09-22 at 10 44 40 PM](https://user-images.githubusercontent.com/8597300/191899134-61d82b24-dbdf-4fde-ac3a-f9ed204c5a37.jpg)

But I also think the button that @pawelru proposed looks good too. 

@kumamiao do you have any opinion on how which button that makes the most sense to the user?

Reply:
Also, @kumamiao, can I also hear your opinion on the wording 'On/Off' or 'Enabled/Disabled' for the button toggle? Which one do you think user will prefer?

Reply:
I am a fan of Pawel's proposition of the On/Off switch for Active filtering which is quite straightforward
![image](https://user-images.githubusercontent.com/22608476/192032035-e34ac8fd-49ce-4515-b7ea-84b12b97eb97.png)
The current button can be a bit misleading and takes more space